masonry/src/widget/textbox.rs

```diff
@@ -170,6 +171,7 @@ impl Widget for Textbox {
                         ctx.request_layout();
                         ctx.request_paint();
                         ctx.request_focus();
+                        ctx.widget_state.needs_window_origin = true;
```

Comment:
I don't know if this is the right way. I am surprised @PoignardAzur had nothing to say about it so far. :+ )

Reply:
It's not the right way. I'm not sure I understand the logic here.